tekton/src/pipelines/fvt-assist.yml.j2

```diff
@@ -78,6 +78,18 @@ spec:
         - input: "$(params.mas_app_channel_assist)"
           operator: notin
           values: [""]
+    - name: imagescan
+      {{ lookup('template', 'taskdefs/ivt-core/common/taskref.yml.j2') | indent(6) }}
+      params:
+        {{ lookup('template', 'taskdefs/ivt-core/common/params.yml.j2') | indent(8) }}
+        - name: product_id
+          value: ibm-mas
```

Comment:
product_id needs to be updated too ... you can get the product_id value from the entry for ivtcore-xxx

Also need to add entry for product_channel same as we have for ivtcore too  (and check it's not missing for the core one while we're doing this) ... 

```
        - name: product_channel
          value: $(params.mas_app_channel_assist)
```